cirque/capabilities/xvnccapability.py

```diff
@@ -24,16 +24,19 @@
 class XvncCapability(BaseCapability):
     X_SOCKET_PATH = '/tmp/.X11-unix'
 
-    def __init__(self, localhost=True, display_id=0):
+    def __init__(self, localhost=True, display_id=0, docker_display_id=None):
```

Comment:
why do we need add this new docker_display_id parameter?
can we not just using display_id in docker?

Reply:
For our use case, the display id inside docker container always has to be 0. If we only use display_id, it means display id 0 has to be always available which might not be the case if we are bringing up more than 1 device.